Hymns/Display/DisplayHymnToolbarView.swift

```diff
@@ -0,0 +1,62 @@
+import SwiftUI
+
+struct DisplayHymnToolbarView: View {
+    @State var toolbarTab = "lyrics"
```

Comment:
I think it might be good to make this an enum. That way in that little if/else clause, we can just use a switch statement and have compile-time assurance that it's exhaustive

Reply:
I couldn't figure this out :/ Whenever, I use the enum cases with the Webview it does not reload when toolbarTab refreshes. I tried some solutions from https://jasonzurita.com/swiftui-if-statement/ with the same results. Also, by using the enum we have to type erase the views since they are not all webviews (ake lyrics). By doing this I read we are also sacrificing some performance since SwiftUI does not know the view type upfront. Idk if this is also a consideration to not go the switch route. I did move the string to an enum though.

Reply:
Yeah, I think you are right. Let's keep it as if/else then!